client/components/modal.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import AppContext from '../lib/app-context';
+
+export default class PostModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      postPicture: 'placeholder.png',
+      location: '',
+      caption: ''
+    };
+    this.fileInputRef = React.createRef();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleCreatePost = this.handleCreatePost.bind(this);
+  }
```

Comment:
This component isn't needed for logging in. In the future, it's best to avoid turning in extra code that doesn't address the work needed. It makes pull requests much harder for the rest of your team to review when there's 2x more than is necessary.